develop/dev-guide-sample-application-java-jdbc.md

```diff
@@ -74,7 +74,7 @@ Connect to your TiDB cluster depending on the TiDB deployment option you've sele
 
 3. Ensure the configurations in the connection dialog match your operating environment.
 
-    - **Endpoint Type** is set to `Public`
+    - **Connection Type** is set to `Public`
```

Comment:
```suggestion
    - **Endpoint Type** is set to `Public`
```
For TiDB Serverless clusters, it is "Endpoint Type" in the console.
<img width="710" alt="image" src="https://github.com/user-attachments/assets/c8b5d0e8-497e-4477-a468-b1c8c6400459">